frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -230,7 +230,7 @@ function ReaderFooter:setupTouchZones()
             id = "readerfooter_tap",
             ges = "tap",
             screen_zone = footer_screen_zone,
-            handler = function() return self:onTapFooter() end,
+            handler = function(ev) return self:onTapFooter(ev) end,
```

Comment:
nitpick, can we keep the variable name as `ges` instead of `ev` to keep it consistent with the rest of the code like readerpaging?